lib/imu.js

```diff
@@ -368,20 +373,25 @@ var Drivers = {
                 // grab the calibration
                 computed.calibration = state;
 
-                // if (state === 0xFF) {
-                  // change this so that we wait for the system
-                  // to be ready, not each component.
-                  // as per adafruit example suggestions
-                  // wonder if the calibration should be checked here, or where we are consuming?
-                if (s === 3) {
+                // emit the calibration state so we can work out in our userspace if
+                // we are good to go, and for when we are performing the calibration steps
+                this.emit("data", computed);
```

Comment:
Cool, I'll look at how this works and  update this and the various subcomponents handling of this.